### PR TITLE
OBBI alpine artifacts

### PR DESCRIPTION
**Pull request by hasithakn**
Issue link: https://github.com/wso2-enterprise/financial-open-banking/issues/8146

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Comment by hasithakn** [comment]
This needs to merge after https://github.com/wso2/docker-open-banking/pull/154

File: docker-compose/mysql/scripts/setup-apim4.0.0-IS5.11.0.sql [hunk 1]
```diff
@@ -10,753 +10,749 @@ openbank_openbankingdb
...
+    )ENGINE INNODB;
 
 CREATE TABLE IF NOT EXISTS IDN_OAUTH2_DEVICE_FLOW_SCOPES (
-            ID INTEGER NOT NULL AUTO_INCREMENT,
-            SCOPE_ID VARCHAR(255),
-            SCOPE VARCHAR(255),
-            PRIMARY KEY (ID),
-            FOREIGN KEY (SCOPE_ID) REFERENCES IDN_OAUTH2_DEVICE_FLOW(CODE_ID) ON DELETE CASCADE
-)ENGINE INNODB;
+                                                             ID INTEGER NOT NULL AUTO_INCREMENT,
```
Comment (Ashi1993) [inline docker-compose/mysql/scripts/setup-apim4.0.0-IS5.11.0.sql:215]: Seems to be formatting issue. Please check

Comment (hasithakn) [reply docker-compose/mysql/scripts/setup-apim4.0.0-IS5.11.0.sql:215]: Ack, @Chathuratharushka please note, as these changes are on top of https://github.com/wso2/docker-open-banking/pull/154 PR

Comment (Chathuratharushka) [reply docker-compose/mysql/scripts/setup-apim4.0.0-IS5.11.0.sql:215]: Fixed it by https://github.com/wso2/docker-open-banking/pull/154/commits/335f4a89ef0c9c7ecf0e5d74bbea5523fef7c643

**Comment by hasithakn** [comment]
Replaced with https://github.com/wso2/docker-open-banking/pull/156

**Pull request closed without merging**